CurrentTimeApp/CurrentTimeApp.csproj

```diff
@@ -8,7 +8,7 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="7.0.1" />
+		<PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="7.0.2" />
```

Comment:
This change has been included in https://github.com/jjliggett/CurrentTimeApp/pull/96 which updates the dependencies and the lock file.